guestlink.php

```diff
@@ -0,0 +1,93 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Page to grant external users access to a BBB session
+ *
+ * @package    mod_bigbluebuttonbn
+ * @author     Angela Baier
+ * @copyright  2020 University of Vienna
```

Comment:
I am sorry. While we appreciate the contribution. This is a no-no for us. You can keep the authorship, but you would need to give away the copyright in favour of  * @copyright 2010 onwards, Blindside Networks Inc

Reply:
Nevermind, I misunderstood this with licencing. It seems to be Ok as it is. You may need to sign a CLA though before we merge, but that is a different story.

I think I will need to do serious changes to UI, but in principle your implementation is outstanding. Thanks again.